CONTRIBUTING.md

```diff
@@ -30,52 +52,96 @@ For more information specific to how a quarto site works, and for examples of wh
+If you spot a problem with the site, [search if an issue already
+exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments).
+If a related issue doesn't exist, you can open a new issue using a
+relevant [issue
+form](https://github.com/dfe-analytical-services/analysts-guide/issues/new/choose).
 
 #### Solve an issue
 
-Scan through our [existing issues](https://github.com/dfe-analytical-services/analysts-guide/issues) to find one that interests you. If you find an issue to work on, you are welcome to open a PR with a fix.
+Scan through our [existing
+issues](https://github.com/dfe-analytical-services/analysts-guide/issues)
+to find one that interests you. If you find an issue to work on, you are
+welcome to open a PR with a fix.
 
 ### Make Changes
 
-Key things to remember when making or proposing changes:
-- Where guidance already exists elsewhere it should be linked to rather than duplicated
-- When adding images, save them in the /images folder
-- When adding files for download, save them in the /resources folder
-- Don't commit rendered .html files for pages
-- If your changes remove or edit a pre-existing anchor link, consider how that will be redirected for users who may have bookmarked it
+Key things to remember when making or proposing changes: - Where
+guidance already exists elsewhere it should be linked to rather than
+duplicated - When adding images, save them in the /images folder - When
+adding files for download, save them in the /resources folder - Don't
+commit rendered .html files for pages - If your changes remove or edit a
+pre-existing anchor link, consider how that will be redirected for users
+who may have bookmarked it
 
 #### Make changes in the UI
 
-Click **Edit this page** at the bottom of the right hand table of contents of any page to make small changes such as a typo, sentence fix, or a broken link. This takes you to the `.Qmd` file where you can make your changes and [create a pull request](#pull-request) for a review.
+Click **Edit this page** at the bottom of the right hand table of
+contents of any page to make small changes such as a typo, sentence fix,
+or a broken link. This takes you to the `.Qmd` file where you can make
+your changes and [create a pull request](#pull-request) for a review.
 
 #### Make changes locally
 
-1. Clone or fork the repository.
+1.  Clone or fork the repository.
 
-2. Open the repository in your editor of choice, e.g. R Studio.
+2.  Open the repository in your editor of choice, e.g. R Studio.
 
-3. Create a working branch and start with your changes!
+3.  Create a working branch and start with your changes!
 
-4. Commit and push the changes to your working branch.
+4.  Commit and push the changes to your working branch.
 
-### Pull Request
+### Pull Request {#pull-request}
```

Comment:
Was this intentionally added?

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/c9e2762b-2ec7-4fd5-8f6b-acd37b4b3dff)